quarkus/config-api/src/main/java/org/keycloak/config/HostnameOptions.java

```diff
@@ -10,11 +10,21 @@ public class HostnameOptions {
             .description("Hostname for the Keycloak server.")
             .build();
 
+    public static final Option HOSTNAME_URL = new OptionBuilder<>("hostname-url", String.class)
+            .category(OptionCategory.HOSTNAME)
+            .description("Set the base URL for frontend URLs.")
```

Comment:
not sure if we should make it more precise that this is an alternative to setting `hostname` and other options in this description (as @vmuzikar mentioned yesterday). 

e.g. `Set the complete base URL for frontend URLs, including scheme, host, port and path.`